packages/csharp/PortingAssistant/PortingAssistant.Api/Program.cs

```diff
@@ -44,11 +44,12 @@ static void Main(string[] args)
                 .MinimumLevel.Debug()
                 .WriteTo.RollingFile(
                     Path.Combine(args[2], "logs", "portingAssistant-assessment-{Date}.log"),
-                    outputTemplate: outputTemplate)
+                    outputTemplate: outputTemplate,
+                    fileSizeLimitBytes: 1000000)
                 .WriteTo.Logger(lc => lc.MinimumLevel.Error().WriteTo.RollingFile(
-                    Path.Combine(args[2], "logs", "portingAssistant-backend-{Date}.log"), 
-                    outputTemplate: outputTemplate)
-                    );
+                    Path.Combine(args[2], "logs", "portingAssistant-backend-{Date}.log"),
+                    outputTemplate: outputTemplate,
+                    fileSizeLimitBytes: 1000000));

```

Comment:
Wanted to confirm the decision on this number.
According to Amazon API Gateway, HTTP API payload size is capped at 10MB.
https://docs.aws.amazon.com/apigateway/latest/developerguide/limits.html
I don't think we want to use all 10MB for the payload, do you know how much payload would this number mean during upload? Considering our history, would this number making a lot of log files instead? Just want to confirm the thought process here in case we run into too many log files issues after this change?

Reply:
A couple things, 1. this is 1 MB, but 2. this is more for consistency on log file size, we still cap each upload at 1000 lines.

The uploader changes behind this will cap the folder at 250 MB so it doesn't matter if we generate more files.